### PR TITLE
Llama2 70 b OOM fix

### PR DESCRIPTION
**Pull request by 12010486**
# Fix
Without this small fix, there is an OOM issue with multiple models.
To reproduce it:

```
cd optimum-habana &&
pip install -e . &&
cd optimum-habana/examples/text-generation &&
pip install -r requirements.txt && 
pip install -r requirements_lm_eval.txt &&
HF_DATASETS_TRUST_REMOTE_CODE=true QUANT_CONFIG=optimum-habana/examples/text-generation/quantization_config/maxabs_measure.json  TQDM_DISABLE=1 python3 ../gaudi_spawn.py --use_deepspeed --world_size 2 run_lm_eval.py --model_name_or_path meta-llama/Llama-2-70b-hf --warmup 0 --use_hpu_graphs -o test_results_measure.json --bf16 --batch_size 8 --use_kv_cache --trim_logits --attn_softmax_bf16 --bucket_size=128 --bucket_internal --trust_remote_code --tasks hellaswag lambada_openai piqa winogrande
```
Accuracy is the same

### CODE REVIEW

**Review by regisss — APPROVED** [review]
LGTM
